src/ui/components/Library/Navigation/TeamButton.tsx

```diff
@@ -49,18 +51,11 @@ export function TeamButton({
         )}
         onClick={onClick}
       >
-        <span className="overflow-hidden whitespace-pre overflow-ellipsis">
-        
-        {
-          isTest == false ? 
-        (  
-          <Icon filename="other" size="medium" className="mr-1 bg-iconColor" />
-        ) 
-        : 
-          <Icon filename="tests" size="medium" className="mr-1 bg-iconColor" />
-        }
-
+        <span className="overflow-hidden whitespace-pre overflow-ellipsis">        
+        <div className="flex">        
+        <LibraryIcon teamType={isTest ? ("tests") : ("team")} />  
```

Comment:
Can simplify this to `<LibraryIcon teamType={isTest ? "tests" : "team"} />  `

Reply:
Fixed!